etl-federation/existing-prometheus/primary-federator.yaml

```diff
@@ -37,23 +31,34 @@ serviceAccount: # this example uses AWS IRSA, which creates a service account wi
   create: false
   name: kubecost-irsa-s3
 prometheus:
-  kubeStateMetrics:
-    enabled: false
-  kube-state-metrics:
-    disabled: true
-  nodeExporter:
-    enabled: false
   server:
     global:
       external_labels:
         # cluster_id should be unique for all clusters and the same value as .kubecostProductConfigs.clusterName
+        # this must be set, even when prometheus is disabled.
         cluster_id: CLUSTER_NAME
+global:
+  grafana: # prometheus metrics will be local cluster only, disable grafana to save resources
+    enabled: false
+    proxy: false
+  prometheus:
+    enabled: false
+    fqdn: http://prometheus-operator-prometheus.monitoring:9090 # local prometheus endpoint
+  thanos: # this is "thanos" but any prometheus compatible service should work below
+    queryService: http://thanos-query-frontend.thanosnamespace.svc:9090 # the location of the aggregated metrics. For thanos, thanos-query-frontend is recommended
+# serviceMonitor: # if your prometheus supports serviceMonitors, set these to true. Otherwise see extraScrapeConfigs.yaml in this directory
+#   enabled: true
+#   additionalLabels:
+#     label-key: label-value
+#   networkCosts:
+#     enabled: true
+#     additionalLabels:
+#       label-key: label-value
 networkCosts:
-  # optional, see: https://docs.kubecost.com/install-and-configure/advanced-configuration/network-costs-configuration
   enabled: true
```

Comment:
Should we let it turn on by default as it require write permission on the cluster ?

Reply:
great question. 
With the idea on this repo being that the SE team will guide on configuration options, it is a small concern.

We need to weigh that against not getting the full value of Kubecost.